corehq/util/mixin.py

```diff
@@ -37,3 +41,15 @@ class ValidateModelMixin(object):
     def save(self, *args, **kwargs):
         self.full_clean()
         super(ValidateModelMixin, self).save(*args, **kwargs)
+
+
+class DisabledDbMixin(object):
```

Comment:
i had moved this over to corehq.util.mixin from corehq.form_processor.models because i started off using it, but then i opted to instead take the approach used in the new `PartitionedModel`. i suppose i'll just leave this here though, since it's still a generic class

@snopoke fyi